packages/@tinacms/fields/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 */
 
-import * as React from 'react'
+import * as React, { SyntheticEvent } from 'react'
```

Comment:
```suggestion
import * as React from 'react'
```
destructuring syntax doesn't work w/ `import * as`